less/index.less

```diff
@@ -57,3 +57,185 @@ html, body {
     height: 100%;
 }
 
+/*variables*/
+@font-color: dimgrey;
+@med-size-font: 2rem;
+
+/*mixins*/
+.font-varela(){
+	font-family: 'Varela Round', sans-serif;
+} 
+.flex-mixin(){
+	display: flex;
+	justify-content: space-between;
+}
+
+/*this helps me figure out the boudaries*/
+*{
+	// border: 1px solid grey !important;
+}
+
+/*styling header*/
+header{
+	.flex-mixin();
+	color: @font-color;
+	background: silver;
+	padding: 2.5%;
+	font-size: 16px;
+
+	@media screen and (max-width: 500px){
+		height: 45%;
+		font-size: 1rem;
+		
+	}
+	.intro{
+		width: 45%;
+		display: flex;
+		flex-direction: column;
+		
+		
+		h1{
+			font-size: 4rem;
+			height: 30%;
+			@media screen and (max-width: 500px){
+				font-size: 2rem;
+			}/*close media query*/
+		}/*closes h1*/
```

Comment:
These comments to let you know when the styles of an element start and end are ok to leave in your code and should be left in so that anyone else looking at the code can read it easier so good job with this!